web3/app.html

```diff
@@ -0,0 +1,265 @@
+  console.log("Loading window")
+
+  const walletButton = document.querySelector("#walletButton")
+  const balanceLabel = document.querySelector("#balanceLabel")
+  const priceLabel = document.querySelector("#priceLabel")
+  const pizzasLabel = document.querySelector("#pizzasLabel")
+  const buyButton = document.querySelector("#buyButton")
+
+  // Placeholders
+  balanceLabel.innerHTML = balance + " Ξ"
+  priceLabel.innerHTML = price + " ₿"
+  pizzasLabel.innerHTML = maxSupply
+
+  initWeb3()
+
+  walletButton.addEventListener('click', function () {
+    console.log("Wallet button pressed")
+    promptMetamask()
+  })
+
+  buyButton.addEventListener('click', function () {
+    handleUser()
+    if (addresses.length == 0) {
+      promptMetamask()
+    }
+    else {
+      triggerPurchase()
+    }
+  })
+
+})
+
+function initWeb3() {
+
+  if (window.ethereum) {
+    console.log("Window.ethereum exists")
+    window.web3 = new Web3(window.ethereum)
+    startApp()
+  } else if (window.web3) {
+    console.log("Window.web3 exists")
+    window.web3 = new Web3(window.web3.currentProvider)
+    startApp()
+  } else if( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) ) {
+    console.log("Mobile initiated")
+    startApp()
+  } else {
+    console.log("Non-ethereum browser detected")
+    window.alert('Non-Ethereum browser detected. Try Chrome and MetaMask!')
+  }
+}
+
+function promptMetamask() {
+  window.ethereum.enable()
+    .then(() => {
+      updateBalance()
+    }).catch((err) => {
+    console.log(err);
+  })
+}
+
+async function triggerPurchase() {
+  Erc721Instance.methods.getPrice().call()
+  .then(function(amount){
+    console.log("priceWei: ", amount)
+    Erc721Instance.methods.purchase().send({from: address, value: amount})
+    .on('receipt', function(receipt){
+      console.log("receipt: ", receipt)
+      window.alert('Transaction confirmed, enjoy your 🍕!')
+      updateValues()
+    })
+    .on('error', function(error, receipt) {
+        console.log("Transaction failed: ", error, "br/", receipt)
+    })
+  })
+  .catch(function(error){
+    console.log("getPrice failed")
+  })
+}
+
+async function startApp() {
+  console.log("App started")
+  Erc721Instance = new web3.eth.Contract(ERC721ABI,ERC721ADDRESS)
+
+  Erc721Instance.events.Transfer(function(error, event){ console.log(event); })
+  .on('data', function(event){
+      console.log("event: ", event)
+      updateValues()
+      //console.log("eventTokenId: ", event["returnedValues"]["tokenId"]) // TO-DO: get pizza # from here
+  })
+  .on('changed', function(event){
+      console.log("changed")
+  })
+  .on('error', console.error);
+
+  updateValues()
+  handleUser()
+}
+
+async function updateValues() {
+  Erc721Instance.methods.getPrice().call()
+  .then(function(amount){
+    price = web3.utils.fromWei(amount)
+    console.log("price: ", price)
+    priceLabel.innerHTML = price + " ₿"
```

Comment:
Same here. Super confusing to display BTC if users are actually transfering ETH Ξ.